client/components/App.js

```diff
@@ -5,13 +5,15 @@ import "../assets/scss/main.scss"
 import { Route, Switch, BrowserRouter } from "react-router-dom"
 import PetTypeList from "./PetTypeList"
 import PetsByType from "./PetsByType"
+import PetShow from "./PetShow"
 
 const App = props => {
   return (
     <div className="callout primary">
       <BrowserRouter>
         <Switch>
           <Route exact path="/pet-types/:id" component={PetsByType} />
+          <Route exact path="/pets/:id" component={PetShow} />
           <Route exact path="/pet-types" component={PetTypeList} />
```

Comment:
I would recommend keeping the routes together by name. /pet-types/:id and /pet-types should be together.